cmd/wallet/address.go

```diff
@@ -74,7 +74,7 @@ func buildNewAddressCmd(parentCmd *cobra.Command) {
 	parentCmd.AddCommand(newAddressCmd)
 
 	addressType := newAddressCmd.Flags().String("type",
-		wallet.AddressTypeBLSAccount, "the type of address: bls_account or validator")
+		wallet.AddressTypeEd25519Account, "the type of address: bls_account, ed25519_account and validator")
```

Comment:
I think it's not common way to use ed25519 as default. Actually it always need password and that's not user friendly. But as you said, it's faster and may convince us to use it as default. In total I'm not agree with this change.

Reply:
I agree. Passwords can be a good reason to consider them harder to use. I prefer to set BLS as default and Ed25519 as recommended. any ideas?

> By recommended I mean just adding a `recommended` tag in the help command.

Reply:
Agree with @akbariandev We need to test it properly.

Reply:
@kehiy revert the first part, just update the help string.